src/app/features/theme-drawer/theme-list-item.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback } from 'react';
+import { Box, BoxProps, color, Flex, Stack } from '@stacks/ui';
+
+import { Title } from '@app/components/typography';
+import { useDrawers } from '@app/common/hooks/use-drawers';
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { UserSelectedTheme, useThemeSwitcher } from '@app/common/theme-provider';
+import { CheckmarkIcon } from '@app/components/icons/checkmark-icon';
+
+interface ThemeListItemProps extends BoxProps {
+  theme: UserSelectedTheme;
+}
+export const ThemeListItem = ({ theme, ...props }: ThemeListItemProps) => {
+  const { setIsShowingThemeSwitcher } = useDrawers();
+  const [_currentTheme, userSelectedTheme, setUserSelectedTheme] = useThemeSwitcher();
+  const isActive = theme === userSelectedTheme;
+  const analytics = useAnalytics();
+
+  const handleItemClick = useCallback(() => {
+    void analytics.track('theme_selected');
+    setUserSelectedTheme(theme);
+    setTimeout(() => setIsShowingThemeSwitcher(false), 25);
+  }, [analytics, setUserSelectedTheme, theme, setIsShowingThemeSwitcher]);
+
+  return (
+    <Box
+      width="100%"
+      key={theme}
+      _hover={
+        isActive
+          ? undefined
+          : {
+              backgroundColor: color('bg-4'),
+            }
+      }
+      px="loose"
+      py="base"
+      onClick={isActive ? undefined : handleItemClick}
+      cursor={isActive ? 'default' : 'pointer'}
+      {...props}
+    >
+      <Flex width="100%" justifyContent="space-between" alignItems="center">
+        <Stack>
```

Comment:
Typically `Stack` is used for stacking more than one element, unless I am missing something I think this is just one?

Reply:
It is, will remove.